cmd/juno/juno.go

```diff
@@ -83,7 +83,9 @@ func main() {
 	config := new(node.Config)
 	cmd := NewCmd(config, func(cmd *cobra.Command, _ []string) error {
 		fmt.Printf("%s\n\n", greeting)
-
+		// Log the version number
+		fmt.Printf("Version: %s\n", Version)
+		
```

Comment:
Let's not log the versions here, instead let's use [spew](https://pkg.go.dev/github.com/davecgh/go-spew/spew) to pretty print the config being printed on line `node/node.go:165`.